notebook/supermusr-plotting-frames.ipynb

```diff
@@ -103,7 +119,7 @@
     "fig, ax = plt.subplots()\n",
     "\n",
     "for frame, channel in zip(frame_data_list, channels):\n",
-    "    line = ax.plot(frame) # Only expecting one line back\n",
+    "    line = ax.plot(x_axis_time, frame) # Only expecting one line back\n",
```

Comment:
Could this be made to fall back to the sample index if the sample rate is not present in the data file?

This way old data can still be viewed.